opal/mca/btl/tcp/btl_tcp_component.c

```diff
@@ -729,7 +732,9 @@ static int mca_btl_tcp_component_create_instances(void)
         char* if_name = *argv;
         int if_index = opal_ifnametokindex(if_name);
         if(if_index < 0) {
-            BTL_ERROR(("invalid interface \"%s\"", if_name));
+            opal_show_help("help-mpi-btl-tcp.txt", "invalid if_inexclude",
```

Comment:
Not really. If I correctly remember the matching of opal_show_help the %s will match if_name, so for eth0 you will get 
```
WARNING: An invalid value was given for btl_tcp_if_eth0.  This
value will be ignored.
```
Not really what we want.